core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.js

```diff
@@ -40,6 +41,7 @@ module.exports = class baseUser {
   async openBrowser() {
     await puppeteer
       .launch({
+        headless: false,
```

Comment:
@ashish-patwal Please add a todo here to investigate the issues with headless mode as discussed in the team groupchat.

Reply:
Yeah sure

Reply:
Done !